misc/xml-to-record-converter/src/test/resources/ballerina/sample_6.bal

```diff
@@ -1,5 +1,5 @@
 type Item record {
-    int ItemCode;
+    (int|string) ItemCode;
```

Comment:
Do we need these parenthesis ?
```suggestion
    int|string ItemCode;
```

Reply:
At the moment, this is how the code is being generated, with the parenthesis, If normally the union types doesn't has parans, we can remove this in upcoming changes, as this will be a big change to do at this time

Reply:
Ack